nil/internal/params/protocol_params.go

```diff
@@ -64,21 +71,26 @@ const (
 	JumpdestGas   uint64 = 1     // Once per JUMPDEST operation.
 	EpochDuration uint64 = 30000 // Duration between proof-of-work epochs.
 
-	CreateDataGas         uint64 = 200   //
-	CallCreateDepth       uint64 = 1024  // Maximum depth of call/create stack.
-	ExpGas                uint64 = 10    // Once per EXP instruction
-	LogGas                uint64 = 375   // Per LOG* operation.
-	CopyGas               uint64 = 3     //
-	StackLimit            uint64 = 1024  // Maximum size of VM stack allowed.
-	TierStepGas           uint64 = 0     // Once per operation, for a selection of them.
-	LogTopicGas           uint64 = 375   // Multiplied by the * of the LOG*, per LOG transaction. e.g. LOG0 incurs 0 * c_txLogTopicGas, LOG4 incurs 4 * c_txLogTopicGas.
+	CreateDataGas   uint64 = 200  //
+	CallCreateDepth uint64 = 1024 // Maximum depth of call/create stack.
+	ExpGas          uint64 = 10   // Once per EXP instruction
+	LogGas          uint64 = 375  // Per LOG* operation.
+	CopyGas         uint64 = 3    //
+	StackLimit      uint64 = 1024 // Maximum size of VM stack allowed.
+	TierStepGas     uint64 = 0    // Once per operation, for a selection of them.
+	LogTopicGas     uint64 = 375  // Multiplied by the * of the LOG*, per LOG transaction.
+	//                               e.g. LOG0 incurs 0 * c_txLogTopicGas, LOG4 incurs 4 * c_txLogTopicGas.
 	CreateGas             uint64 = 32000 // Once per CREATE operation & contract-creation transaction.
 	Create2Gas            uint64 = 32000 // Once per CREATE2 operation
 	SelfdestructRefundGas uint64 = 24000 // Refunded following a selfdestruct operation.
-	MemoryGas             uint64 = 3     // Times the address of the (highest referenced byte in memory + 1). NOTE: referencing happens on read, write and in instructions such as RETURN and CALL.
-
-	TxDataNonZeroGasFrontier  uint64 = 68   // Per byte of data attached to a transaction that is not equal to zero. NOTE: Not payable on data of calls between transactions.
-	TxDataNonZeroGasEIP2028   uint64 = 16   // Per byte of non-zero data attached to a transaction after EIP 2028 (part in Istanbul)
+	MemoryGas             uint64 = 3     // Times the address of the (highest referenced byte in memory + 1).
+	//                                      NOTE: referencing happens on read,
+	//                                      write and in instructions such as RETURN and CALL.
```

Comment:
Let's put the comments above variables

Reply:
![image](https://github.com/user-attachments/assets/aa77279b-3017-4e38-90cb-a0bc4d833a0f)
What confuses me is that the IDE then tells me that the comments should be styled differently. This is not a mechanical change that can be done without thinking (and without knowing the code). Our linters don't check such things, so we can forget about it. What do you think is the best thing to do? Should I still make all comments over variables?

Reply:
Yes. Because right now this part is formatted in a quite ugly way.

Reply:
I agree, I thought long and hard about what to do here too. I did it the way you suggested.